### PR TITLE
UI changes to make it similar to production

### PR DESCRIPTION
**Pull request by smialko**
Small UI changes discussed in comment with Michael here: https://trello.com/c/alrOo9tK/1814-display-sessions-fixed

### CODE REVIEW

File: app/javascript/react/components/Map/Markers/SingleMarker/SingleMarker.style.tsx [hunk 1]
```diff
@@ -53,16 +53,16 @@ const DataContainer = styled.div`
...
+  height: 1.2rem;
   border-radius: 50%;
   background-color: ${(props) => props.color};
   pointer-events: none;
 `;
 
 const MarkerText = styled(H4)`
   color: ${gray400};
-  font-size: 1.2rem;
+  font-size: 12px;
```
Comment (evulins) [inline app/javascript/react/components/Map/Markers/SingleMarker/SingleMarker.style.tsx:65]: Is it different size from 1.2rem?

**Review by evulins — APPROVED** [review]

File: app/javascript/react/components/Map/Markers/SingleMarker/SingleMarker.style.tsx (hunk 1 above)
Comment (smialko) [reply app/javascript/react/components/Map/Markers/SingleMarker/SingleMarker.style.tsx:65]: According to chatgpt yes 🤔

Comment (smialko) [reply app/javascript/react/components/Map/Markers/SingleMarker/SingleMarker.style.tsx:65]: Should I change it?

**Review by jankrzempek — COMMENTED** [review]
Everything perfect! One thing I am not sure about, is the outline of the session. currently, on prod each "marker" has an outline of the color that matches the threshold and Michael says in the comment: "when the outline is removed, the visual contrast between the markers and the grey default map is reduced, which to me doesn’t look good". How we approach this?

File: app/javascript/react/components/Map/Markers/SingleMarker/SingleMarker.style.tsx (hunk 1 above)
Comment (evulins) [reply app/javascript/react/components/Map/Markers/SingleMarker/SingleMarker.style.tsx:65]: If it looks the same i would keep 1.2rem if not 12px

**Comment by smialko** [comment]
@jankrzempek , the latest Michael's comment about outline: please retain the feature of the new marker design that alters the halo of the selected session. this looks good and is an effective way to indicate a relation between the graph and the selected marker.

**Pull request merged**